src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int firstSide;
+    private int secondSide;
+    private int height;
+
+    public IsoscelesTrapezoid(int firstSide, int secondSide, int height, Color color) {
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+        this.height = height;
+        super.color = color;
+    }
+
+    @Override
+    public double getArea() {
+        return ((firstSide + secondSide) * height / 2);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println(

```

Comment:
Avoid using direct print statements. You should rather override the toString() method to provide a string representation of the object.

Reply:
You are fully right, but if I understood correctly the instruction of the exercise it is forbidden.